src/witan/models/dem/ccm/models.clj

```diff
@@ -17,7 +17,7 @@
 
    ;; asfr/asmr projections
    [:calc-hist-asfr :project-asfr]
-   [:calc-hist-asmr :project-asmr]
+   [:calc-hist-asmr :project-asmr-average-fixed]
```

Comment:
this doesn't need to change in the workflow - only in the fn that is pointed to in the tasks. I'm pretty sure - can you have a go at changing that and see if it still works?